python/hsfs/feature_view.py

```diff
@@ -783,6 +1301,77 @@ def train_test_split(
         """
         Get training data from feature groups.
 
+        !!! example "Create random splits"
+            ```python
+            # get feature store instance 
+            fs = ...
+
+            # get feature view instance 
+            feature_view = fs.get_feature_view(...)
+
+            # get training data
+            X_train, X_test, y_train, y_test = feature_view.train_test_split(
+                test_size=0.2
+            )
+
+            ```
+
+        !!! example "Create time-series based splits"
+            ```python
+            # get feature store instance 
+            fs = ...
+
+            # get feature view instance 
+            feature_view = fs.get_feature_view(...)
+
+            # set up dates
+            train_start = "2022-05-01 00:00:00"
+            train_end = "2022-06-04 23:59:59"
+            test_start = "2022-07-01 00:00:00"
+            test_end= "2022-08-04 23:59:59"
+            # you can also pass dates as datetime objects
+
+            # get training data
+            X_train, X_test, y_train, y_test = feature_view.train_test_split(
+                test_size=0.2,
+                train_start=train_start,
+                train_end=train_end,
+                test_start=test_start,
+                test_end=test_end,
+                description='Description of a dataset'
+            )
+            ```
+        !!! warning "Label Encoder can fail while creating training splits. Feature view is just a logical union of feature groups which stores a metadata. While creating dataset splits - label encoder fits on train split. There may be unique values in the test data. As a result Label Encoder fails in handling unique unfitted values."            
+            ```python
+            import pandas as pd
+
+            df = pd.DataFrame({
+                'category_col':['category_a','category_b','category_c','category_d'],
+                'numeric_col': [40,10,60,40]
+            })
+
+            feature_group = fs.get_or_create_feature_group(
+                name='feature_group_name',
+                version=1,
+                primary_key=['category_col']
+            )
+
+            feature_group.insert(df)
+
+            label_encoder = fs.get_transformation_function(name='label_encoder')
+
+            feature_view = fs.create_feature_view(
+                name='feature_view_name',
+                query=feature_group.select_all(),
+                transformation_functions={'category_col':label_encoder}
+            )
+
+            feature_view.train_test_split(
+                test_size=0.5
+            )
+            # Output: KeyError: 'category_a'
+            ```
+
         !!! info
```

Comment:
Again, remove this info box, please.

Reply:
Please remove this info box from all the places it is duplicated in this file. There's like 7 places in this file with the same info box.